### DISCUSSION TITLE
Implementing Azure AD B2C in a Giraffe App

### DISCUSSION THREAD

**Discussion by scorelocity** [giraffe-fsharp/Giraffe#558] [Q&A]
Is there a sample of implementing authentication in a Giraffe app using Azure AD B2C? I have tried to do so by following tutorials written for C# and translating to F#/Giraffe but have not been able to get this to work. Part of the problem seems to be that MS provides some UI in Microsoft.Identity.Web.UI that may not work in Giraffe.

**Comment by 64J0** [comment]
+1, I think this would be great
